mods/sgssb/moves.js

```diff
@@ -615,7 +615,8 @@ exports.BattleMovedex = {
 		id: "aquasubscribe",
 		name: "Aqua Subscribe",
 		priority: 1,
-		self: {
+		secondary: {
```

Comment:
I think youll need to specify `target: "self"` in the secondary so it dosent boost the foe.